### PR TITLE
add some packages license tags

### PR DESCRIPTION
**Pull request by feckert**
Maintainer: @champtar 
Compile tested: only license tag updates
Run tested: only license tag updates

Description:
Update some Missing license tags

### CODE REVIEW

File: utils/cryptodev-linux/Makefile [hunk 1]
```diff
@@ -17,6 +17,7 @@ PKG_SOURCE_URL:=https://github.com/cryptodev-linux/cryptodev-linux
 PKG_SOURCE_PROTO:=git
 PKG_SOURCE_VERSION:=a705360197260d28535746ae98c461ba2cfb7a9e
 PKG_MIRROR_HASH:=334a8f4df18e36e97459b4c6669c9873c13c4f5dae25243cf41ff70c2ce661d8
+PKG_LICENSE:=GPL-2.0+
```
Comment (yousong) [inline utils/cryptodev-linux/Makefile]: Looks like it should be only "GPL-2.0" according to content of https://github.com/cryptodev-linux/cryptodev-linux/blob/master/COPYING

Comment (yousong) [reply utils/cryptodev-linux/Makefile]: `PKG_LICENSE_FILES` should also be added.

File: libs/libgcrypt/Makefile [hunk 2]
```diff
@@ -14,6 +14,7 @@ PKG_RELEASE:=1
 PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.bz2
 PKG_SOURCE_URL:=ftp://ftp.gnupg.org/gcrypt/libgcrypt
 PKG_HASH:=f9461b4619bb78b273a88d468915750d418e89a3ea3b641bab0563a9af4b04d0
+PKG_LICENSE:=GPL-2.0+
```
Comment (yousong) [inline libs/libgcrypt/Makefile]: According to https://gnupg.org/software/libgcrypt/index.html, libgcrypt library was licensed under LGPL-2.1+.  Please also add the `PKG_LICENSE_FILES` definition

File: utils/cryptodev-linux/Makefile (hunk 1 above)
Comment (feckert) [reply utils/cryptodev-linux/Makefile]: Sorry this was my vault!
Will fix it

File: libs/libgcrypt/Makefile (hunk 2 above)
Comment (feckert) [reply libs/libgcrypt/Makefile]: I am not sure what the right PKG_LICENSE is!
For the lib it is true it is LGPL-2.1+ but for the helper it is GPL-2.0.
But the package builds lib and bin at once.
What is the right Text to add to PKG_LICENSE?
PKG_LICENSE_FILES is only for the bin GPL-2.0

Comment (yousong) [reply libs/libgcrypt/Makefile]: Yeah, I also had the same concern.  Looks like we can just specify all the licenses in use in `PKG_LICENSE`.  There is already an example doing this in `include/u-boot.mk` of LEDE repo.

**Pull request merged**